src/styles/header.scss

```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
```

Comment:
it's not part of the `header` BEM block - shouldn't be here. Move to a separate file or at least write it in the index file